enterprise/frontend/src/embedding-sdk/config.ts

```diff
@@ -1,2 +1,10 @@
+import packageJson from "embedding-sdk/package.template.json";
```

Comment:
This puts all the json file in the bundle so I was a bit worried, but it's just the template package.json which is small so it's probably not a problem.
<img width="671" alt="image" src="https://github.com/user-attachments/assets/e24873dc-68e2-40b9-abd9-871437f4f921">

If we wanted to optimize this or if the template grows bigger, we could move this import in the webpack config and pass the version as ENV.